_docker/drupal/drupal-filesystem/web/config/sync/field.storage.node.field_cheat_sheet_image.yml

```diff
@@ -0,0 +1,30 @@
+uuid: d9d91bcf-da23-4f80-abbd-ac5f13df3bc9
```

Comment:
We may need to modify this in the future due to the issue of images disappearing.